common/config/rush/common-versions.json

```diff
@@ -18,7 +18,9 @@
      * instead of the latest version.
      */
     // "some-library": "1.2.3"
-    "resolve": "1.11.0"
+    "resolve": "1.11.0",
+    "@azure/ms-rest-azure-js": "1.3.5",
```

Comment:
If `ms-rest-js` and `ms-rest-azure-js` are direct dependencies, we should not add them to this file. This file should only be used when we need to pin a transitive dependency we have no direct control over.

All we should need to do is change our package.json files to simply reflect the version requirement we want, which you appear to already have done in this PR.

The only reason I can think of to change this file is if these libraries ever appear as transitive dependencies of another library that we depend on.